packages/cli/package.json

```diff
@@ -39,6 +39,7 @@
     "test:ci": "pnpm run test"
   },
   "dependencies": {
+    "@ark/util": "catalog:",
```

Comment:
what does `catalog:` do?

Reply:
it's a better version of aligning versions of a given package: https://pnpm.io/catalogs
https://github.com/latticexyz/mud/blob/main/pnpm-workspace.yaml#L5-L11